cpp/Esri/ArcGISRuntime/Toolkit/CoordinateConversionOption.h

```diff
@@ -0,0 +1,248 @@
+/*******************************************************************************
+ *  Copyright 2012-2020 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#ifndef ESRI_ARCGISRUNTIME_TOOLKIT_COORDINATECONVERSIONOPTION_H
+#define ESRI_ARCGISRUNTIME_TOOLKIT_COORDINATECONVERSIONOPTION_H
+
+// Qt headers
+#include <QObject>
+
+// ArcGISRuntime headers
+#include <GeometryTypes.h>
+#include <Point.h>
+#include <SpatialReference.h>
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+
+namespace Toolkit
+{
+
+/*!
+ * \brief a CoordinateConversionOption is a collection of properties that
+ * dictates how a Point should be converted to and from a string.
+ * 
+ * a CoordinateConversionOption is able to convert between Point <--> QString
+ * using the formatting options it is currently set with.
+ */
+class CoordinateConversionOption : public QObject
+{
+  Q_OBJECT
+  Q_CLASSINFO("RegisterEnumClassesUnscoped", "false")
```

Comment:
What does this mean/do?

Reply:
It means in QML Code that enums we declare will have syntax `CoordianteType.Gars` as opposed to defaulting to just `Gars`.

Reply:
That's really nice. Do we use this anywhere else?